mpa/modules/hooks/ib_loss_hook.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from mmcv.runner import Hook, HOOKS
+from mmcv.parallel import is_module_wrapper
+
+
+@HOOKS.register_module()
+class IBLossHook(Hook):
+    def __init__(self, dst_classes):
+        """Hook for IB loss.
+        It passes the number of data per class and current epoch to IB loss class.
+
+        Args:
+            dst_classes (list): A list of classes including new_classes to be newly learned
+        """
+        self.cls_num_list = None
+        self.dst_classes = dst_classes
+
+    def before_train_epoch(self, runner):
+        # get loss from model
+        model_loss = self._get_model_loss(runner)
+
+        # pass the number of data per class and current epoch to IB loss
+        dataset = runner.data_loader.dataset
+        num_data = self._get_num_data(dataset)
```

Comment:
Is this a value that changes every epoch?

Reply:
Maybe no. I just thought of possibility that dataset can be resized after epochs.

Reply:
If so, isn't this an unnecessary operation every epoch?

Reply:
Could you explain more about `dataset can be resized after epochs?` I can't imagine that case :(

Reply:
I think it too far from this implementation. I fixed the codes reflecting all your reviews. Please check again.